src/graphql/query/borrowerSocialShareData.graphql

```diff
@@ -0,0 +1,14 @@
+query borrowerSocialShareData {
+	general {
+		socialShareExperiment: uiExperimentSetting(
+			key: "borrower_social_share"
+		) {
+			key
+			value
+		}
+	}
+	experiment(id: "borrower_social_share") @client {
```

Comment:
This can't be queried for until after the uiExperimentSetting query completes, so this will need to be done in 2 separate queries.